Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -535,7 +535,7 @@ extension ProjectDescription.Target {
                 .replacingOccurrences(of: "-", with: "_"),
             bundleId: target.name
                 .replacingOccurrences(of: "_", with: "-"),
-            deploymentTarget: deploymentTarget,
+            deploymentTargets: [deploymentTarget],
```

Comment:
here we should use multiple deployment targets as well

Reply:
@danyf90 Could you clarify why we need to support multiple deployments targets here? I tried to add deployment targets for each platform in package (.iOS, .macOS, .tvOS, .watchOS] but I doubt that it is correct way.
```swift
extension ProjectDescription.DeploymentTarget {
    fileprivate static func from(
        platform: ProjectDescription.Platform,
        minDeploymentTargets: [ProjectDescription.Platform: ProjectDescription.DeploymentTarget],
        package: [PackageInfo.Platform],
        packageName _: String
    ) throws -> [Self] {
        guard !package.isEmpty else { return [minDeploymentTargets[platform]!] }
        
        var deploymentTargets = [Self]()
        
        for packagePlatform in package {
            // Deployment targets below the minimum one raises warnings
            let targetVersion = try Self.max(packagePlatform.version, minDeploymentTargets[platform]?.targetVersion)

            switch ProjectDescription.Platform(rawValue: packagePlatform.platformName) {
            case .iOS:
                let hasMacCatalyst = package.contains(where: { $0.platformName == "maccatalyst" })
                deploymentTargets.append(.iOS(
                    targetVersion: targetVersion,
                    devices: hasMacCatalyst ? [.iphone, .ipad, .mac] : [.iphone, .ipad]
                )
                                         )
            case .macOS:
                deploymentTargets.append(.macOS(targetVersion: targetVersion))
            case .watchOS:
                deploymentTargets.append(.watchOS(targetVersion: targetVersion))
            case .tvOS:
                deploymentTargets.append(.tvOS(targetVersion: targetVersion))
            default:
                break
            }
        }
        
        return deploymentTargets
    }
```

Reply:
Let's say you have a multi platform iOS+macOS target depending on an external package, shouldn't the target coming from the package be iOS+macOS as well?

Reply:
Also, right now [we are creating a target for each platform](https://github.com/tuist/tuist/blob/45b1c71bb008aaa86751de6c9ec3789fc9fd6c03/Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift#L383-L401), while we should create a single multi-platform target

Reply:
@danyf90 What do you think about naming multiplatform target? In my opinion single platform target should has platform prefix but Multiplatform target shouldn't.

Reply:
yes, I think we can remove prefixes at all for single multiplatform target & single platform target, can't we?